### ISSUE TITLE
Cannot read property 'map' of null

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#23691]
Found this in insiders after doing many theme changes:

```
Cannot read property 'map' of null: TypeError: Cannot read property 'map' of null
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/json/client/out/jsonMain.js:69:107
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by aeschli** [comment]
it's fixed already (18affc0b0bca6d6e889a5e1d08b13cfbc79f851d)

**Comment by aeschli** [comment]
Verified by opening a JSON document that has no color information
